frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -355,8 +355,8 @@ function _useKubeObjectLists<K extends KubeObject>({
   for (const cluster of clusters) {
     if (items === null) {
       items = clusterResults[cluster].items;
-    } else {
-      items = items.concat(clusterResults[cluster].items!);
+    } else if (clusterResults[cluster].items !== null) {
+      items = items.concat(clusterResults[cluster].items);
```

Comment:
Maybe `items.concat(clusterResults[cluster].items ?? []);` would be more readable, but either way is fine.

Reply:
Thanks. Fixed.